src/daemon/config.rs

```diff
@@ -303,40 +303,57 @@ pub struct IssuanceTimingConfig {
 pub struct RepositoryRetentionConfig {
     #[serde(default = "RepositoryRetentionConfig::dflt_retention_old_notification_files_seconds")]
     pub retention_old_notification_files_seconds: i64,
-    #[serde(default = "RepositoryRetentionConfig::dflt_retention_delta_files_seconds")]
-    pub retention_delta_files_seconds: i64,
     #[serde(default = "RepositoryRetentionConfig::dflt_retention_delta_files_min_nr")]
     pub retention_delta_files_min_nr: usize,
```

Comment:
These settings don't seem to be documented in `krill-pubd.conf`. They are only present with the `dflt_` prefix, not without it.

Reply:
removed dflt_